Cargo.lock

```diff
@@ -150,9 +150,9 @@ dependencies = [
 
 [[package]]
 name = "anstyle-query"
-version = "1.0.3"
+version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "a64c907d4e79225ac72e2a354c9ce84d50ebb4586dee56c82b3ee73004f537f5"
```

Comment:
Do we need to bump version or we can leave it for a separate PR?

Reply:
Apparently not. I had issues with an old `ahash` version, but cargo clean and reinstall was enough.

Reply:
Removed the changes in 1c6f607e79dd0c8dbbee8c8610bb949c0bd45b6b